epytope/EpitopePrediction/External.py

```diff
@@ -2046,19 +2071,24 @@ def parse_external_result(self, file):
         :return: A dictionary containing the prediction results
         :rtype: dict
         """
-        result = defaultdict(defaultdict)
         f = csv.reader(open(file, "r"), delimiter='\t')
+        scores = defaultdict(defaultdict)
         for r in f:
             if not r:
                 continue
-
+            
             row = r[0].split()
             if not len(row):
                 continue
-
-            if "HLA-" not in row[0]:
+            
+            if "HLA-" not in row[HLAIndex.NETMHCII_2_2]:
```

Comment:
do we have prefix string/enum for alleles?

Reply:
I thought that would also be meaningful. I found hardcoded indices for alleles in 3 classes. Can also remove again if you wish

Reply:
Sorry, this is a misunderstanding. I was referring to "HLA-". It would be better to replace that as well. "HLA" e.g. is already given as field `organism` in the respective Allele class.

Reply:
But that `if` checks only if the the current row contains allele information 🤔. But maybe I misunderstood your point